docs/promscale_alerting.md

```diff
@@ -0,0 +1,226 @@
+    annotations:
+      summary: Slow Promscale ingestion
+      description: "90% of ingestion batch took {{ $value }} seconds to ingest."
+# Query
+  - alert: PromscaleQueryHighErrorRate
+    expr: |
+      (
+        sum by (job, instance, type) (
+          rate(promscale_query_requests_total{code=~"5.."}[5m])
+        )
+      /
+        sum by (job, instance, type) (
+          rate(promscale_query_requests_total[5m])
+        )
+      ) > 0.05
+    for: 1m
+    labels:
+      severity: warning
+      component: connector
+    annotations:
+      summary: High error rate in querying Promscale
+      description: "Evaluating queries via Promscale has {{ $value }} error rate."
+  - alert: PromscaleQueryHighErrorRate
+    expr: |
+      (
+        sum by (job, instance, type) (
+          rate(promscale_query_requests_total{code=~"5.."}[5m])
+        )
+      /
+        sum by (job, instance, type) (
+          rate(promscale_query_requests_total[5m])
+        )
+      ) > 0.1
+    for: 1m
+    labels:
+      severity: critical
+      component: connector
+    annotations:
+      summary: High error rate in querying Promscale
+      description: "Evaluating queries via Promscale had {{ $value }} error rate."
+  - alert: PromscaleQueryHighLatency
+    expr: |
+      (
+        histogram_quantile(
+          0.90,
+          sum by (job, instance, type, le) (
+            rate(promscale_query_duration_seconds_bucket[5m])
+          )
+        ) > 5
+      and
+        sum by (job, instance, type) (
+          rate(promscale_query_duration_seconds_bucket[5m])
+        ) > 0
+      )
+    for: 1m
+    labels:
+      severity: warning
+      component: connector
+    annotations:
+      summary: Slow Promscale querying
+      description: "90% of the queries took {{ $value }} seconds to evaluate."
+  - alert: PromscaleQueryHighLatency
+    expr: |
+      (
+        histogram_quantile(
+          0.90,
+          sum by (job, instance, type, le) (
+            rate(promscale_query_duration_seconds_bucket[5m])
+          )
+        ) > 10
+      and
+        sum by (job, instance, type) (
+          rate(promscale_query_duration_seconds_bucket[5m])
+        ) > 0
+      )
+    for: 1m
+    labels:
+      severity: critical
+      component: connector
+    annotations:
+      summary: Slow Promscale querying
+      description: "90% of the queries took {{ $value }} seconds to evaluate."
+# Cache
+  - alert: PromscaleCacheHighNumberOfEvictions
+    expr: |
+      (
+        sum by (job, instance, name, type) (
+          rate(promscale_cache_evictions_total[5m])
+        )
+      /
+        sum by (job, instance, name, type) (
+          promscale_cache_capacity_elements
+        )
+      ) > 0.2
+    for: 5m
+    labels:
+      severity: warning
+      component: connector
+    annotations:
+      summary: High cache eviction in Promscale
+      description: "Promscale clockcache is evicting at {{ $value }} entries a second."
```

Comment:
```suggestion
      description: "Promscale {{ $commonLabels.name }} is evicting at {{ $value }} entries a second."
```

Let's be future-proof and use templates for the cache name which is problematic.